ui/app/templates/components/auth-info.hbs

```diff
@@ -30,27 +31,21 @@
               </button>
             </li>
             <li class="action">
-              <ConfirmAction
-                @buttonClasses="button link is-destroy"
-                @confirmTitle={{concat "Revoke " (get auth 'authData.displayName') "?"}}
-                @confirmMessage={{concat "You will not be able to log in again with this token."}}
-                @confirmButtonText="Revoke"
-                @onConfirmAction={{action "revokeToken"}}
-              >
-                Revoke token
-              </ConfirmAction>
+              <c.Message
+                @id={{get auth 'authData.displayName'}}
+                @title={{concat "Revoke " (get auth 'authData.displayName') "?"}}
+                @onConfirm={{action "revokeToken"}}
+                @message="You will not be able to log in again with this token."
+                @triggerText={{concat "Revoke " (get auth 'authData.displayName')}}/>
```

Comment:
Should this still be "Revoke token" ? (and same with the one below)

Reply:
yes! thank you for noticing that.